pulp_rpm/app/migrations/0044_noartifact_modules.py

```diff
@@ -1,67 +1,55 @@
 # Generated by Django 3.2.15 on 2022-08-12 16:28
 
+import os
 from django.db import migrations, models, transaction
 
 
 def convert_artifact_to_snippets(apps, schema_editor):
     """Create snippet from artifact and remove artifact."""
     with transaction.atomic():
-        Modulemd = apps.get_model('rpm', 'Modulemd')
-        ModulemdDefaults = apps.get_model('rpm', 'ModulemdDefaults')
-        ContentArtifact = apps.get_model('core', 'ContentArtifact')
-        Artifact = apps.get_model('core', 'Artifact')
+        Modulemd = apps.get_model("rpm", "Modulemd")
+        ModulemdDefaults = apps.get_model("rpm", "ModulemdDefaults")
+        ContentArtifact = apps.get_model("core", "ContentArtifact")
         modules_with_snippet = []
         defaults_with_snippet = []
-        content_artifacts_to_delete = []
-        artifacts_to_delete = []
 
         for module in Modulemd.objects.all():
             artifact = module._artifacts.get()
-            module.snippet = artifact.file.read().decode("utf-8")
-            content_artifact = ContentArtifact.objects.get(artifact__pk=artifact.pk)
-            content_artifacts_to_delete.append(content_artifact.pk)
-            artifacts_to_delete.append(artifact.pk)
+            if os.path.exists(artifact.file.path):
```

Comment:
@pavelpicka We may need to go back and adjust this to work on S3 / Azure based installations